taas/config.d/organizations.yml

```diff
@@ -4,30 +4,67 @@ sources:
         options:
             build_by_default: False
         beta-v1:
-            url: https://docs.google.com/spreadsheets/d/1oa7Wknz64dMQlV2XsW0ssjh6Gw5SnHVfGxDSw_5eKOQ/edit#gid=518938746
+            url: https://docs.google.com/spreadsheets/d/1GIA3FrRM1GZS3FoUDNpw35MhC_WLTdKBiiU2tUlWS84/edit#gid=0
             mapping:
-                id: HR info ID
-                label: Preferred Term
-                acronym:
-                    type: map
-                    field: ACRONYM
+                id: ID
+                hrinfo_id:
+                    field: HRinfo ID
+                    optional: True
+                rw_id:
+                    field: RW ID
                     optional: True
-                homepage:
-                    type: map
-                    field: homepage
+                ocha_gms_id:
+                    field: OCHA_GMS ID
                     optional: True
-                fts_id:
-                    type: map
-                    field: FinancialTrackingService ID
+                label.default: Preferred Term
+                label.humanitarianresponse:
+                    field:
+                        - HRinfo Alt Term
+                        - Preferred Term
+                label.gms:
+                    field:
+                        - GMS Alt Term
+                        - Preferred Term
+                label.reliefweb:
+                    field:
+                        - RW Alt Term
+                        - Preferred Term
+                label.fts:
+                    field:
+                        - FTS Alt Term
+                        - Preferred Term
+                used_for:
```

Comment:
Hooray! This makes much more sense as a top-level field now we know it's different from the other labels. And now the other labels are all consistent with their defaulting behaviour. :)